pages/common/kubeadm.md

```diff
@@ -0,0 +1,31 @@
+# kubeadm
+
+> Command line interface for creating and managing Kubernetes clusters.
+
+- Create a Kubernetes master node:
+
+`kubeadm init`
+
+- Bootstrap a Kubernetes worker node and join it to a cluster:
+
+`kubeadm join --token {{token}}`
+
+- Create a new bootstrap token with a TTL of 12 hours:
+
+`kubeadm token create --ttl 12h0m0s`
+
+- Check if the Kubernetes cluster is upgradeable and which versions are available:
+
+`kubeadm upgrade plan`
+
+- Upgrade Kubernetes cluster to a specified version:
+
+`kubeadm upgrade apply {{version}}`
+
+- View the kubeadm ConfigMap containing the cluster's configuration:
```

Comment:
Any reason to write ConfigMap in CamelCase ?

Reply:
Yes -- Kubernetes documentation typically writes it that way when referring to a ConfigMap object (e.g. [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) and [here](https://kubernetes.io/docs/tasks/administer-federation/configmap/))